src/main/java/com/weavers/duqhun/business/impl/ProductServiceImpl.java

```diff
@@ -360,6 +373,7 @@ public CartBean getCartFoAUser(Long userId) {
             //=====================ready product bean====================//
             for (ProductSizeColorMap sizeColorMap : sizeColorMaps) {
                 ProductBean productBean = new ProductBean();
+                productBean.setQty("1");

```

Comment:
It is denote as min quantity. when ProductBean return from user this qty will be same as user set.